corehq/warnings.py

```diff
@@ -37,6 +37,8 @@
     ("tastypie", "request.is_ajax() is deprecated"),
     ("nose.suite", "'collections.abc'"),
     ("nose.plugins.collect", "'collections.abc'"),
+    ("packaging.version",
+     "Creating a LegacyVersion has been deprecated and will be removed in the next major release"),
```

Comment:
Is this still needed? What code is triggering it?

Reply:
yes, based on this [build failure](https://github.com/dimagi/commcare-hq/actions/runs/3905755829/jobs/6673141312), `module: packaging.version line 111`

Reply:
It looks like the line that triggered that build failure has changed since that build. It now uses `LooseVersion('')` rather than `parse_version('')`:

https://github.com/dimagi/commcare-hq/blob/eb52f8224a38a7a8fa5bbfd742da625accace41a/corehq/apps/app_manager/feature_support.py#L10

I think this may not be needed anymore. Although it's possible that `LooseVersion` is also creating a `LegacyVersion` when passed `''`, which would be unfortunate. If that is the case then I think we should find a way to make that not happen, by passing a different value, for example.

Reply:
ah yes, thanks! removing this line now...